src/components/playButton/PlayButton.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { Icon } from '../icon'
+import { space } from '../../theme/space'
+import { PlayButtonProps } from './PlayButton.types'
+import { ButtonWrapper } from './PlayButton.styles'
+
+export const PlayButton: React.FC<PlayButtonProps> = () => {
```

Comment:
Following making the aforementioned update to render the component as a `button` instead of a `div`, you can update the props here.

React has a built-in type for `button` props so you no longer need to have a custom interface here. The type is called `React.ButtonHTMLAttributes<HTMLButtonElement>` and could be utilised as:

```
export const PlayButton: React.FC<React.ButtonHTMLAttributes<HTMLButtonElement>> = (props) => {
  // component
}
```